src/components/Logo/index.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
 import configuration from 'shared/services/configuration/configuration';
 
+const StyledLogo = styled.img`
+  width: ${props => props.tall === true ? configuration.logo.width : configuration.logo.smallWidth};
```

Comment:
Or 
```jsx
width: ${({ tall }) => tall ? configuration.logo.width : configuration.logo.smallWidth};
```

Reply:
I don't think so. You could potentially set `100%` as value in the configuration.

Reply:
Of course, I see. Than we need to make sure to add that to the configuration file.

Reply:
Will check if `jest-styled-components` will understand this and adjust it if so.

Reply:
It does :)

Reply:
Do we want this flexibility though? It was pixels up till now and that is enough I would say.
Otherwise you move css syntax to the configuration.

Reply:
@boris-arkenaar I don't really understand what you mean, before my changes there was no `px` value hard coded in the template strings, this is the same after this changes, since this value is a string we could use pixels or percentage.

Or I missed something ;)

Reply:
Before the value was used in the `width` and `height` attributes of the `img` tag which assumes pixel values by default. Now you use them in css attributes which mean you would need to add `px`.

So if you add `px` to the code, no need to change the config.

If you don't add `px` to the code, the configuration needs to be updated. The configuration becomes more powerful, but does it need to?

Reply:
@boris-arkenaar check, I missed something! I did not realize (or forgot) that the `pixels` were set automatically on a `HTML attribute`, whoops!, thanks for clarifying!

That said, we are using percentages for those values so would :heart: to keep the **flexibility**, hope you can live with that.

Reply:
I agree with @jpoppe on this one; we indeed don't want our CSS rules to be reflected by the application's configuration, but having [length](https://developer.mozilla.org/en-US/docs/Web/CSS/length) values instead of numerical values greatly increases the flexibility for the positioning of the application's logo.